articles/openshift/oshift-ref-self-service-api.md

```diff
@@ -0,0 +1,190 @@
+---
+title: UKCloud for OpenShift API Reference Guide | UKCloud Ltd
+description: Shows how to interact with UKCloud for OpenShift via use of an application programming interface (API)
+services: openshift
+author: Sue Highmoor
+toc_rootlink: Reference
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: OpenShift API Reference Guide
+toc_fullpath: Reference/oshift-ref-self-service-api.md
+toc_mdlink: oshift-ref-self-service-api.md
+---
+
+# UKCloud for OpenShift API Reference Guide
+
+## Overview
+
+In your UKCloud for OpenShift cluster, there are some operations that you need to perform on the underlying OpenStack infrastructure, such as verifying the status of the VMs underpinning the cluster, or rebooting a node. To help you accomplish these operations, OpenShift provides a REST API.
```

Comment:
It'd be good to be clear this isn't an OpenShift API but a UKCloud API that overlays some elements of the OpenShift and OpenStack API's.
```suggestion
In your UKCloud for OpenShift cluster, there are some operations that you need to perform on the underlying OpenStack infrastructure, such as verifying the status of the VMs underpinning the cluster, or rebooting a node. To help you accomplish these operations, UKCloud provide a REST API.
```